### PR TITLE
Delete the progress bar from flow B.

### PR DESCRIPTION
**Pull request by svillafe**
# Delete the progress bar from flow B

## Why?
Surprisingly after we ran the first A/B Test, the old design performed better than the new design. Therefore we decide to remove the progress bar from the new design since our hypothesis is that it distracting the final user.

## Changes
* `_membership.css`: In order to delete the progress bar, we modify the margin and paddings of the heading.
* `register-page-membership.hbs`: Removed the progress bar from the HTML.
* `signin-page-membership.hbs`: Removed the progress bar from the HTML.
* `main.css`: Removed the progress bar file.
* Deleted all the files related to the progress bar.

## Related PRs

* CSS changes for Flow B at membership-frontend: https://github.com/guardian/membership-frontend/pull/1403
* Turning on the A/B test in membership-frontend: https://github.com/guardian/membership-frontend/pull/1404

### Desired
new-fow-bw 1

### Final result

screencapture-profile-thegulocal-register-1481105205900

### CODE REVIEW

File: public/signin-page-membership.hbs [hunk 1]
```diff
@@ -11,22 +11,15 @@
 
   <h1 class="page-heading">{{ signInPageText.title }}</h1>
 
-  <section class="signin__progress-bar-mem">
```
Comment (SiAdcock) [inline public/signin-page-membership.hbs]: If we kill these elements, can we also kill the corresponding CSS for `.signin__progress-bar-mem` and `.signin__bottom-progress-bar-mem`?

Comment (svillafe) [inline public/signin-page-membership.hbs]: Done :)

**Comment by svillafe** [comment]
It is working on *CODE*.
code-working-signup

code-working-singin

**Comment by svillafe** [comment]
It is working on **PROD**.

prod-sigin
prod-signup

**Pull request merged**